demo-app/src/main/java/cash/z/ecc/android/sdk/demoapp/demos/listtransactions/ListTransactionsFragment.kt

```diff
@@ -48,12 +48,18 @@ class ListTransactionsFragment : BaseDemoFragment<FragmentListTransactionsBindin
         // have the seed stored
         val seed = Mnemonics.MnemonicCode(seedPhrase).toSeed()
 
-        initializer = runBlocking {
-            Initializer.new(requireApplicationContext()) {
-                runBlocking { it.newWallet(seed, network = ZcashNetwork.fromResources(requireApplicationContext())) }
-                it.setNetwork(ZcashNetwork.fromResources(requireApplicationContext()))
+        initializer = Initializer.newBlocking(
+            requireApplicationContext(),
+            Initializer.Config {
+                runBlocking {
```

Comment:
Pacu, this is the actual fix for the SDK initialization with an existing birthday.  The change was the `newWallet()` -> `importWallet()`